war/src/main/less/modules/app-bar.less

```diff
@@ -0,0 +1,32 @@
+.jenkins-app-bar {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 30px;
+  padding-bottom: 30px;
```

Comment:
I would suggest this could be changed to `2rem`. All `30px` instances really.

Reply:
Done :)